package.json

```diff
@@ -45,7 +45,7 @@
     "concurrently": "^8.0.1",
     "fast-glob": "^3.3.1",
     "husky": "^8.0.3",
-    "lerna": "^7.3.1",
+    "lerna": "^7.4.1",
```

Comment:
Is the Lerna ^7.3.1  related to the scanning problem?

Reply:
no, it was just a dependency update as it might be related to the issue.

Reply:
OK